libfaketime/__init__.py

```diff
@@ -103,17 +104,22 @@ def __init__(self, datetime_spec, only_main_thread=True):
             _datetime = dateutil.parser.parse(datetime_spec)
 
         self.time_to_freeze = _datetime  # freezegun compatibility
-        self.libfaketime_spec = _datetime.strftime('%Y-%m-%d %T %f')
 
     def _should_fake(self):
         return self.only_main_thread and threading.current_thread().name == 'MainThread'
 
+    def _format_datetime(self, _datetime):
+        return _datetime.strftime('%Y-%m-%d %T %f')
+
+    def tick(self, delta=datetime.timedelta(seconds=1)):
+        self.time_to_freeze += delta
+        os.environ['FAKETIME'] = self._format_datetime(self.time_to_freeze)
```

Comment:
Oh, wow, that's tricky; nice work looking into this.

> Since directly accessing the tick attribute in freezegun is an undocumented behaviour, I personally see no harm in having these small differences between both libraries

Yeah, definitely agree with this :+1: